js/ap_calc.js

```diff
@@ -424,6 +424,7 @@ function Field() {
     var isLightScreen = [$("#lightScreenL").prop("checked"), $("#lightScreenR").prop("checked")];
     var isForesight = [$("#foresightL").prop("checked"), $("#foresightR").prop("checked")];
     var isHelpingHand = [$("#helpingHandR").prop("checked"), $("#helpingHandL").prop("checked")]; // affects attacks against opposite side
+    var isFriendGuard = [$("#friendGuardR").prop("checked"), $("#friendGuardL").prop("checked")];
```

Comment:
You have L and R reversed here. See the comment above this line - only Helping Hand affects damage *against* the side opposite to the one checked.